lib/ret_web/controllers/api/v1/media_search_controller.ex

```diff
@@ -54,4 +64,32 @@ defmodule RetWeb.Api.V1.MediaSearchController do
   def index(conn) do
     conn |> send_resp(422, "")
   end
+
+  defp user_assets_index(conn, account, %{"source" => "assets"} = params) do
+    {:commit, results} =
+      %Ret.MediaSearchQuery{source: "assets", user: account.account_id, type: "all", cursor: params["cursor"] || 1} |> Ret.MediaSearch.search()
```

Comment:
if you pull from master you'll see the cursor is now handled differently, you'll need to fall back to passing `"1"` here